lib/srv/sess.go

```diff
@@ -676,7 +676,7 @@ func (s *session) startInteractive(ch ssh.Channel, ctx *ServerContext) error {
 		if err != nil {
 			return trace.Wrap(err)
 		}
-		s.recorder, err = events.NewAuditWriter(events.AuditWriterConfig{
+		rec, err := events.NewAuditWriter(events.AuditWriterConfig{
```

Comment:
Is this bug more exacerbated by the improved `Cache.GetRemoteCluster`, or just something that came up while improving test reliability?

Reply:
Unrelated panic that was cropping up in tests.